insights/parsers/systemd/unitfiles.py

```diff
@@ -69,15 +56,99 @@ def parse_content(self, content):
                 self.services[service] = enabled
                 self.parsed_lines[service] = line
                 self.service_list.append(service)
+            elif any(part in valid_states for part in parts):
+                self.service_details['ACTIVE'] = parts[2]
+                self.service_details['SUB'] = parts[3]
+                if any(unit in parts[0] for unit in valid_units):
+                    self.service_details['UNIT'] = parts[0]
+                    self.unit_list[parts[0]] = self.service_details
+                self.service_details = {}
```

Comment:
Should `service_details` be a local variable instead of a class attribute?  It appears you only use it to temporarily hold information until you add it to `unit_list`.